### PR TITLE
Update logging info in node operator manual

### PR DESCRIPTION
**Pull request by arshdeeptinna**
Update node operator manual to point out that log files is rotated after every restart so users may need to check latest tar files to troubleshoot any issues
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: doc/node-operator.md [hunk 1]
```diff
@@ -277,7 +277,7 @@ You can use nohup to run the script in detached mode:
 
 ### View Logs
 
-Logs are written to `qsp-protocol-node.log` file in .qsp-protocol directory within the user home directory.
+Logs are written to `qsp-protocol-node.log` file in .qsp-protocol directory within the user home directory. Logs are rotated on each restart of the node. So in case the node is restarted because of any issues check the latest tar files for troubleshooting the underlying cause
```
Comment (alex-murashkin) [inline doc/node-operator.md]: Could we also clarify the naming pattern? And be more explicit on what the node operators should do to send us all the logs?

File: doc/node-operator.md [hunk 2]
```diff
@@ -287,7 +287,7 @@ Or run the node in the background(detached mode) using the following command:
 
 ### View Logs
 
-Logs are written to `qsp-protocol-node.log` file in .qsp-protocol directory within the user home directory.
+Logs are written to `qsp-protocol-node.log` file in .qsp-protocol directory within the user home directory. Logs are rotated on each restart of the node. So in case the node is restarted because of any issues check the latest tar files for troubleshooting the underlying cause. In case you open a github issue attach the log files or paste relevant log messages in the issue.
```
Comment (alex-murashkin) [inline doc/node-operator.md]: `GitHub`.
And let's add very specific instructions. Should they zip the entire folder? If so, let's mention it, or maybe even add a command for this.

File: doc/node-operator.md [hunk 3]
```diff
@@ -286,8 +286,17 @@ Or run the node in the background(detached mode) using the following command:
...
-Logs are written to `qsp-protocol-node.log` file in .qsp-protocol directory within the user home directory.
+To find all log files:
+1. cd to ${HOME}/.qsp-protocol directory
+1. run `ls -lhtr`
+1. Check the creation time of the log files
+1. untar the latest log file to tmp dir using `tar -xvf <tar-file-name> -C /tmp`
+1. Check the file for relevant error messages 
+
+You may have to follow the above process multiple times to debug an issue. 
+If you open up a github issue, please include your findings and [attach all relevant log files].(https://help.github.com/en/articles/file-attachments-on-issues-and-pull-requests)
```
Comment (alex-murashkin) [inline doc/node-operator.md]: `GitHub`

**Comment by martinetlee** [comment]
Do we also want to ask them to provide their `event_database.db` as part of the log?

**Comment by alex-murashkin** [comment]
Yeah, @arshdeeptinna could you document the event_database.db file?

**Pull request merged**